scripts/create_strings.js

```diff
@@ -1,18 +1,18 @@
-var ios_script = require('./create_ios_strings');
 var android_script = require('./create_android_strings');
+var ios_script = require('./create_ios_strings');
 
 module.exports = function (context) {
     var platforms = context.opts.platforms;
 
     var promises = [];
 
-    if (platforms.indexOf('ios') >= 0) {
-        promises.push(ios_script(context));
-    }
-
     if (platforms.indexOf('android') >= 0) {
         promises.push(android_script(context));
     }
 
+    if (platforms.indexOf('ios') >= 0) {
```

Comment:
not sure why this re-order was necessary ... thinking

Reply:
Not necessary but alpha order as much as common order in Cordova plugins to deal with Android before iOS.